Index: pymssql-2.1.4/setup.py
===================================================================
--- pymssql-2.1.4.orig/setup.py
+++ pymssql-2.1.4/setup.py
@@ -156,8 +156,6 @@ SYSTEM = platform.system()
 
 print("setup.py: platform.system() => %r" % SYSTEM)
 print("setup.py: platform.architecture() => %r" % (platform.architecture(),))
-if SYSTEM == 'Linux':
-    print("setup.py: platform.linux_distribution() => %r" % (platform.linux_distribution(),))
 if SYSTEM != 'Windows':
     print("setup.py: platform.libc_ver() => %r" % (platform.libc_ver(),))
 
